src/components/notifications/Cellsynt.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <div class="mb-3">
+        <label for="cellsynt-login" class="form-label">{{ $t("Username") }}</label>
+        <input id="cellsynt-login" v-model="$parent.notification.cellsyntLogin" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="cellsynt-key" class="form-label">{{ $t("Password") }}</label>
```

Comment:
Is password the correct way to authorise against this provider? (most providers have api-keys for automated systems)

Reply:
Yes. They not using any API-keys. They use username and password. 
Check the documentation here: https://www.cellsynt.com/pdf/Cellsynt_SMS_gateway_HTTP_interface_(English).pdf